sentences/de/_common.yaml

```diff
@@ -54,6 +54,9 @@ expansion_rules:
   schliessen: "(schließ | schließe)"
   machen: "(mach | mache)"
   setzen: "(setze | stelle | ändere)"
+  verringere: "(verringere | reduziere)"
+  erhoehe: "(erhöhe | steigere)"
```

Comment:
I would suggest to merge `verringere` (_decrease_) and `erhoehe` (_increase_) into `setzen` (_set_).
This would also lower the complexity of possible sentenc definitions, without really loosing any noteworthy  grammatical correctness.

Reply:
Good point, i changed that.